src/ray/core_worker/lib/java/org_ray_runtime_task_NativeTaskSubmitter.cc

```diff
@@ -156,7 +156,7 @@ Java_org_ray_runtime_task_NativeTaskSubmitter_nativeCreateActor(
   ray::ActorID actor_id;
   auto status =
       GetCoreWorker(nativeCoreWorkerPointer)
-          .CreateActor(ray_function, task_args, actor_creation_options, &actor_id);
+          .CreateActor(ray_function, task_args, actor_creation_options, "", &actor_id);
```

Comment:
```suggestion
          .CreateActor(ray_function, task_args, actor_creation_options, /* extension_data=*/"", &actor_id);
```

Reply:
I added comments to empty extension data.